style/haskell/sample.hs

```diff
@@ -1,3 +1,6 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+
```

Comment:
I think that it would be good to be consistent. I prefer the whitespace, but you've written and read more Haskell than I have, so I'll defer to you.

Updated with the space removed and a guideline not to include it.